### ISSUE TITLE
Custom nginx configuration

### ISSUE THREAD

**Issue by cytopia** [cytopia/devilbox#75]
@ilsenem continued from here: https://github.com/cytopia/devilbox/issues/73

> Another one. Problem was with max upload size in first place to let me upload large dump to MySQL through adminer. PHP was the first gate and now nginx get in the way. Do i have to change configuration inside .devilbox/etc/* folder or there is an easy way?

Yes, unfortunately for now, this is the only place to adjust HTTPD configuration.
Could you let me know what values you need to change.

**Edit**:

For now you should use `.devilbox/etc/<server>/00-defaults.conf`

I am also about to re-write the HTTPD servers so configurations can be done in the `cfg/` directory

**Comment by ilsenem** [comment]
Configuration should look like this:

```
server {
    ...
    client_max_body_size <value>m;
    ...
}
```
If it is possible to move such frequently modified values to `.env` file, it would be great! Some `Vagrant` boxes, like `Homestead` have settings similar to mentioned above.

**Comment by cytopia** [comment]
@ilsenem just to give you some heads-up. This is currently under development.

**Comment by degoya** [comment]
how is the progress on this? looking for custom nginx configuration for each project setup.

**Comment by cytopia** [comment]
Hi guys, to give you some heads up, it is still under development, but shouldn't been too long until its finished (~4 weeks).

I am currently refactoring and modulizing everything at https://github.com/devilbox and put each tool in separate directories so it is either for people to contribute and find errors.

Current work involves making the customization of vhosts work as well as basing all webservers on the official docker container rather than building them from scratch.

**Comment by cytopia** [comment]
To keep you guys up to date with the progress. Listing available/open/closed pull request for that feature.

* [X] nginx stable: https://github.com/cytopia/docker-nginx-stable/pull/6
* [ ] nginx mainline
* [ ] apache 2.2
* [ ] apache 2.4

**Comment by cytopia** [comment]
Is going to be included in the upcoming release: https://github.com/cytopia/devilbox/pull/130

**Comment by cytopia** [comment]
@ilsenem @degoya Could you give it a try and report back if it works as expected for you.

**Comment by cytopia** [comment]
@ilsenem @degoya can you check out this documentation part and see if it is easy to understand how to change the configuration: https://github.com/cytopia/devilbox/blob/c87201db13b43de0dc73098602f1357788f6a886/docs/Configure.md#433-customize-the-vhost-configuration

**Comment by cytopia** [comment]
Merged and ready in v0.12
